### PR TITLE
[AAE-10089] Process variables values are not shown for old tasks and …

### PR DESCRIPTION
**Pull request by Qetikv**
…processes

**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [X] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-10089

**What is the new behaviour?**

Process variables values are shown for old tasks and processes

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by BSekula — APPROVED** [review]

File: lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts [hunk 1]
```diff
@@ -56,23 +56,23 @@ export class VariableMapperService {
...
-        const columnsByVariableId = columnsSchema
+        const columnsByVariables = columnsSchema
             .filter(column => !!column.customData)
             .reduce<{ [variableId: string]: string }>((columnsByVariable, column) => {
                 const columnTitle = column.title;
-                const variableIds = column.customData.assignedVariableDefinitionIds;
+                const variables = column.customData.variableDefinitionsPayload;
 
-                variableIds.forEach((variableId) => {
+                variables.forEach((variableId) => {
```
Comment (BSekula) [inline lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts:68]: please change `variableId`

File: lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts [hunk 2]
```diff
@@ -56,23 +56,23 @@ export class VariableMapperService {
...
     }
 
-    private mapColumnKeysByVariableId(
+    private mapColumnKeysByVariable(
         columnsSchema: DataColumn<ProcessListDataColumnCustomData>[]
     ): { [variableId: string]: string } {
-        const columnsByVariableId = columnsSchema
+        const columnsByVariables = columnsSchema
             .filter(column => !!column.customData)
             .reduce<{ [variableId: string]: string }>((columnsByVariable, column) => {
```
Comment (BSekula) [inline lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts:64]: please change `variableId `

File: lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts (hunk 1 above)
Comment (Qetikv) [reply lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts]: Changed

File: lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts (hunk 2 above)
Comment (Qetikv) [reply lib/process-services-cloud/src/lib/services/variable-mapper.sevice.ts]: changed

**Pull request merged**